blackbox/docs/sql/administration/user_defined_functions.txt

```diff
@@ -232,10 +232,71 @@ the JavaScript function should return a ``GeoJson`` object or``WKT`` string::
 
 .. NOTE::
 
-   If the return value of the JavaScript function is ``undefined`` it is converted
-   to ``NULL``.
+   If the return value of the JavaScript function is ``undefined`` it is
+   converted to ``NULL``.
 
 
+Working with ``TIMESTAMP``
+..........................
+
+Nashorn, the JavaScript interpreter that we use, interprets numbers as
```

Comment:
[Nashorn](http://www.oracle.com/technetwork/articles/java/jf14-nashorn-2126515.html) (link), the JavaScript engine CrateDB uses, ...